Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -76,8 +77,8 @@ bool AVIDump::Start(int w, int h)
 {
   s_pix_fmt = AV_PIX_FMT_RGBA;
 
-  s_width = w;
-  s_height = h;
+  s_width = g_Config.iDumpWidth ? g_Config.iDumpWidth : w;
```

Comment:
An alternative to stretching the image out would be to use the libavfilter library to do a "centered pan-and-scan" type thing (similar to the "Crop" option). We'd set up a filter graph that crops the incoming image to the right aspect ratio (e.g. `crop="'if(gt(a,8/3), ih*8.0/3.0, iw)':'if(gt(a,8/3), ih, iw*3/8)'"` for an 8:3 AR) and then scales it to the appropriate size. That'd also replace the current direct usage of libswscale for pixel format changes (it's done automatically by filter graphs as-needed).